### ISSUE TITLE
Include rollover images

### ISSUE THREAD

**Issue by codl** [codl/mastodon-apod#108]
Some APoD posts have two images, one that's shown by default and one that's shown on rollover, usually one with labels overlaid. Right now the bot only posts the first image, it would be nice to include both in the post

Example: <https://apod.nasa.gov/apod/ap220420.html> has
![](https://apod.nasa.gov/apod/image/2204/PlanetBridge_Honda_960.jpg)
![](https://apod.nasa.gov/apod/image/2204/PlanetBridge_Honda_960_unann.jpg)

**Comment by codl** [comment]
In that particular example, the 2nd image url comes from an onmousover attribute on the link

```html
<a href="image/2204/PlanetBridge_Honda_3000.jpg" 
    onmouseover="if (document.images)
document.imagename1.src='image/2204/PlanetBridge_Honda_960_unann.jpg';"
    onmouseout="if (document.images)
document.imagename1.src='image/2204/PlanetBridge_Honda_960.jpg';">
    <img name="imagename2" src=... />
</a>
```

I need to check how standard that is, if it's always like that then a simple regex will work.

If not uhh. I can probably cobble something together so i can run the onmouseover script with [dukpy](https://pypi.org/project/dukpy/)

**Comment by codl** [comment]
So one problem I'm running into is that sometimes the labeled image is the first one, sometimes it's the second

However the click-thru image is always not labeled

So I think if I want to support rollover images then I have to ignore the click thru image and just use the thumbnail and the rollover

**Comment by codl** [comment]
oh this was implemented in #120 lol i forgot to tag it
